tests/tensor/test_extra_ops.py

```diff
@@ -404,6 +404,19 @@ def test_axis(self):
 
         assert res.broadcastable == (False, False)
 
+        variable = TensorType(config.floatX, [True, False, True, False, True])()
+        res = squeeze(variable, axis=(0, -1))
+
+        assert res.broadcastable == (False, True, False)
```

Comment:
A little tidbit here, we should make these cases with `pytest.parametrize`

Reply:
`parametrize` makes tests a bit less readable but has the advantage of being a bit quicker to spot the failing condition, and reusing code. The latter is not really a thing here though... Anyway, I would not object to refactoring it :)